src/components/LoggedCabinet/LoggedCabinet.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Please, lowercase the name of the file and the folder

Reply:
Renamed